### PR TITLE
Move js css changes to src files

### PR DESCRIPTION
**Pull request by josebarata**
[#135146467] Move changes to minified files to their respective unminified

### CODE REVIEW

File: service/static/css/muk.funder.min.css [hunk 1]
```diff
@@ -1 +1,344 @@
...
\ No newline at end of file
+
```
Comment (tharglet) [inline service/static/css/muk.funder.min.css:1]: Why has this been unminified?

File: service/static/css/muk.institution.dep.min.css [hunk 2]
```diff
@@ -1,271 +1,271 @@
-.daterangepicker {
```
Comment (tharglet) [inline service/static/css/muk.institution.dep.min.css:1]: Another .min that's big?

File: service/static/css/muk.institution.min.css [hunk 3]
```diff
@@ -1 +1,405 @@
...edges-bs3-n-separate-or-term-selector-container{margin-top:20px;margin-bottom:20px}.muk-institution-report-template-tab-bar{border-bottom:2px solid #eee;text-align:right;line-height:44px;color:#999}.muk-institution-report-template-tab-bar li{margin-bottom:-2px}.muk-institution-report-template-tab-bar li.active>a,.muk-institution-report-template-tab-bar li.active>a:focus,.muk-institution-report-template-tab-bar li.active>a:hover{border-left:2px solid #eee;border-right:2px solid #eee;border-top:2px solid #0e90d2;border-bottom:3px solid #fff}.muk-institution-report-template-tab-container{width:100%;height:auto}.muk-institution-report-template-tab-container .muk-institution-report-template-tab{height:700px}.edges-bs3-tabular-results-table{border-collapse:collapse;text-align:right}.edges-bs3-tabular-results-cell-Institution{text-align:left;white-space:nowrap}#date_range{padding:10px;background:#eee}.edges-bs3-bs-multi-date-range-select{margin:10px 0}.edges-bs3-tabular-results-tablediv{max-height:600px;overflow-y:scroll}#edges-bs3-tabular-results-download-data_table{text-decoration:underline}.edges-bs3-tabular-results-table td,.edges-bs3-tabular-results-table th{border:1px solid #e4e4e4}
```
Comment (tharglet) [inline service/static/css/muk.institution.min.css:1]: Big .min

File: service/static/images/manifest.json [hunk 4]
```diff
@@ -1,41 +1,997 @@
-{
```
Comment (tharglet) [inline service/static/images/manifest.json:1]: Why has this manifest file gone from json to html?!

**Review by tharglet — CHANGES_REQUESTED** [review]
Looks like the minified files have been unminified? 

Unsure where our CSS/JS changes have moved to... mind running me through what bits you've moved where?

File: service/static/css/muk.institution.min.css (hunk 3 above)
Comment (josebarata) [reply service/static/css/muk.institution.min.css:1]: Well check for yourself...https://github.com/JiscMonitor/monitor-uk/tree/a7cedd0d56c220f6475507be7aa5827713e1da9d/service/static/css
I've reverted everything to it's original state

File: service/static/images/manifest.json (hunk 4 above)
Comment (josebarata) [reply service/static/images/manifest.json:1]: Problem on merge. Reverted to proper Json...

**Review by tharglet — APPROVED** [review]

**Pull request merged**